roles/gluster_hci/tasks/ssl-setup.yml

```diff
@@ -0,0 +1,102 @@
+---
+- set_fact:
+      ssl_allow_list: "{{ (groups['all'] + gluster_features_ssl_clients |
+                           default([])) | join(',')}}"
+
+# Generate temporary file for storing the keys
+- name: Create temporary directory to store keys
+  tempfile:
+     state: directory
+     suffix: ssl
+  register: result
+  delegate_to: 127.0.0.1
+  run_once: True
+
+- name: Get the name of the directory created
+  set_fact:
+     ssl_base_dir: "{{ result['path'] }}"
+  run_once: True
+
+- name: Generate private key on the servers
```

Comment:
does this run for all servers in inventory?

Reply:
Yes.